File: src/java/array/ContainsDuplicate.java
package java.array;

import java.util.HashSet;
import java.util.Set;

public class ContainsDuplicate {
    public boolean containsDuplicate(int[] nums) {
        Set<Integer> setColl = new HashSet<>();
        for (int item : nums) {
            if (setColl.contains(item))
                return true;
            setColl.add(item);
        }
        return false;
    }
}

File: src/kotlin/array/ContainsDuplicate.kt


class ContainsDuplicate {
    fun containsDuplicate(nums: IntArray): Boolean {
        val setColl = mutableSetOf<Int>()
        for (item in nums) {
            if(setColl.contains(item))
                return true
            setColl.add(item)
        }
        return false
    }
}